.github/dependabot.yml

```diff
@@ -0,0 +1,10 @@
+---
+version: 2
+updates:
+  - package-ecosystem: "gomod"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    rebase-strategy: "disabled"
+    commit-message:
+      prefix: "vendor"
```

Comment:
use prefix as rebase?

Reply:
A lot of Go projects use `vendor` for this. Neither `rebase` or `vendor` labels exist in this project, so I assumed `vendor` would be suitable?

I want to add a rule to automatically label PRs (with `vendor`) that only include changes to `go.mod`, `go.sum` and `vendor/`.

If you really prefer `rebase` and commit prefix and label, I am fine with that too.

Reply:
am fine with it.